tap4j/src/test/resources/org/tap4j/parser/issueGitHub15/issue-15-tap-stream.tap

```diff
@@ -0,0 +1,10 @@
+    ok 1 - subtest 1a
+    ok 2 - subtest 1b
+    1..2
+ok 1 - Subtest 1
+    ok 1 - subtest 2a
+    ok 2 - subtest 2b
+    1..2
+ok 2 - Subtest 2
+1..2
+ok
```

Comment:
Nice!!! Thinking about using the following TAP stream (based on this one)

```
    ok 1 - subtest 1a
    1..1
ok 1 - Subtest 1
    ok 1 - subtest 2a
    ok 2 - subtest 2b
    1..2
ok 2 - Subtest 2
    not ok 1 - subtest 3a
    1..1
not ok 3 - Subtest 3
1..3
ok
```
And then updating the test (already did it locally, and it worked like a charm).